Jenkinsfile

```diff
@@ -33,7 +33,7 @@ pipeline {
       post {
         always {
           junit 'output/**/*.xml'
-          cobertura autoUpdateHealth: true, autoUpdateStability: true, coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '100, 0, 0', failUnhealthy: true, failUnstable: false, lineCoverageTargets: '100, 0, 0', maxNumberOfBuilds: 0, methodCoverageTargets: '100, 0, 0', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
+          cobertura autoUpdateHealth: false, autoUpdateStability: true, coberturaReportFile: 'coverage.xml', conditionalCoverageTargets: '95, 0, 95', failUnhealthy: true, failUnstable: true, lineCoverageTargets: '95, 0, 95', maxNumberOfBuilds: 0, methodCoverageTargets: '95, 0, 95', onlyStable: false, sourceEncoding: 'ASCII', zoomCoverageChart: false
```

Comment:
There is a separate discussion on this but overall this seems like a bad idea

Reply:
This was a recommendation from infra. See [here](https://conjurhq.slack.com/archives/C8BNMU0KV/p1606741415071000)